_config/config.yml

```diff
@@ -7,7 +7,9 @@ After:
 Director:
   rules:
    'timednotice//$Action/$ID/$OtherID': 'TimedNoticeController'
-
+Page:
+  extensions:
+    - TimedNoticedPageExtension
```

Comment:
Indeed a good point. I changed this and noted in the install instructions two ways to handle this: either adding the extension or (less resource draining) adding it directly to the Page.php